components/prize_pool/wikis/brawlstars/prize_pool_custom.lua

```diff
@@ -0,0 +1,98 @@
+---
+-- @Liquipedia
+-- wiki=brawlstars
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Opponent = require('Module:OpponentLibraries').Opponent
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+-- Brawlstars is still in the process of tier conversion
+-- hence "Monthly" is needed here until that is done
+local TIER_VALUE = {16, 8, 4, 2, Monthly = 2}
+
+local TYPE_MODIFIER = {offline = 1, ['offline/online'] = 0.75, ['online/offline'] = 0.75, default = 0.65}
+
+local HEADER_DATA = {}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+
+	args.allGroupsUseWdl = true
+	args.groupScoreDelimiter = '-'
+
+	local prizePool = PrizePool(args):create()
+
+	HEADER_DATA.tournamentName = args['tournament name']
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	if args['smw mute'] or not Namespace.isMain() or Logic.readBool(Variables.varDefault('disable_LPDB_storage')) then
```

Comment:
Same

Reply:
the current (old) ppt does it like that on brawlstars

Reply:
https://liquipedia.net/brawlstars/index.php?search=insource%3A%22smw+mute%22

Neither is used (disable LPDB storage is used only transfers only)

Reply:
`disable_LPDB_storage` gets ste by the same template as `disable_SMW_storage`
https://liquipedia.net/commons/index.php?title=Template:SMW_storage&action=edit

this is widely used to disable storage (on several wikis)

kicking smw mute stuff

Reply:
Should probably add `disable_LPDB_storage` to the default config of PPT.
`SMW_storage` is not used in any league on Brawl Stars so not needed here

Reply:
alright kicking it